packages/features/bookings/Booker/config.ts

```diff
@@ -54,28 +59,86 @@ export const resizeAnimationConfig: ResizeAnimationConfig = {
   small_calendar: {
     default: {
       width: "calc(var(--booker-meta-width) + var(--booker-main-width))",
+      minHeight: "450px",
+      height: "auto",
       gridTemplateAreas: `"meta main"`,
       gridTemplateColumns: "var(--booker-meta-width) var(--booker-main-width)",
     },
     selecting_time: {
-      width: "100%",
-      maxWidth: "calc(var(--booker-meta-width) + var(--booker-main-width) + var(--booker-timeslots-width))",
+      width: "calc(var(--booker-meta-width) + var(--booker-main-width) + var(--booker-timeslots-width))",
+      minHeight: "450px",
+      height: "auto",
       gridTemplateAreas: `"meta main timeslots"`,
       gridTemplateColumns: "var(--booker-meta-width) 1fr var(--booker-timeslots-width)",
     },
   },
   large_calendar: {
     default: {
-      width: "100%",
+      width: "100vw",
+      height: "100vh",
       gridTemplateAreas: `"meta main"`,
       gridTemplateColumns: "var(--booker-meta-width) 1fr",
     },
   },
   large_timeslots: {
     default: {
-      width: "100%",
+      width: "100vw",
+      height: "100vh",
       gridTemplateAreas: `"meta main"`,
       gridTemplateColumns: "var(--booker-meta-width) 1fr",
     },
   },
 };
+
+/**
+ * This hook returns a ref that should be set on the booker element.
+ * Based on that ref this hook animates the size of the booker element with framer motion.
+ * It also takes into account the prefers-reduced-motion setting, to not animate when that's set.
+ */
+export const useBookerResizeAnimation = (layout: BookerLayout, state: BookerState) => {
+  const prefersReducedMotion = useReducedMotion();
+  const [animationScope, animate] = useAnimate();
```

Comment:
This uses the fairly new hook `useAnimate` from framer motion. That gives us an animate function which we can use to run animations.